src/components/Table/TableHeaderRow.tsx

```diff
@@ -13,7 +13,7 @@ export const TableHeaderRow: FC<TableHeaderRowProps> = ({ item, state, children
     const { rowProps } = useTableHeaderRow({ node: item }, state, ref);
 
     return (
-        <tr {...rowProps} ref={ref} className="tw-py-4 tw-px-8">
+        <tr {...rowProps} ref={ref} className="tw-py-4 tw-px-8 tw-sticky tw-top-0 tw-bg-white tw-z-10">
```

Comment:
Nice!! 🎉  I would just suggest to change the class `tw-bg-white` for `tw-bg-base`

Reply:
Will do! I thought we would need to use our color palette defined in `tailwind.config.js`. Isn't that our design system?

Reply:
We decided to use alias tokens instead of brand tokens. You can find them here: https://fondue-components.frontify.com/?path=/story/tokens--alias-tokens
Usually, you can find, on the Figma file, which token to use. Tokens can be confusing sometimes 😄